### PR TITLE
[TASK] Remove registries and add yaml file loader

### PR DESCRIPTION
**Pull request by NeoBlack**

### CODE REVIEW

File: Classes/DashboardConfiguration.php [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+
+    /**
+     * Short-hand function for the cache
+     *
+     * @return FrontendInterface
+     * @throws NoSuchCacheException
+     */
+    protected function getCache(): FrontendInterface
+    {
+        return GeneralUtility::makeInstance(CacheManager::class)->getCache('core');
```
Comment (haassie) [inline Classes/DashboardConfiguration.php]: This is v10 only. In v9 it is cache_core. Would be great if we can check both in this case.

File: Classes/Controller/WidgetAjaxController.php [hunk 2]
```diff
@@ -13,6 +13,16 @@
...
 class WidgetAjaxController
 {
+    /**
+     * @var DashboardConfiguration
+     */
+    protected $dashboardConfiguration;
+
+    public function __construct(DashboardConfiguration $dashboardConfiguration = null)
+    {
+        $this->dashboardConfiguration = $dashboardConfiguration;
```
Comment (haassie) [inline Classes/Controller/WidgetAjaxController.php]: We need to have a fallback for v9:

```$this->dashboardConfiguration = $dashboardConfiguration ?? GeneralUtility::makeInstance(DashboardConfiguration::class);```

File: Classes/DashboardConfiguration.php (hunk 1 above)
Comment (NeoBlack) [reply Classes/DashboardConfiguration.php]: good catch, I will push an update

File: Classes/Controller/WidgetAjaxController.php (hunk 2 above)
Comment (NeoBlack) [reply Classes/Controller/WidgetAjaxController.php]: also good catch

**Pull request merged**